github/authorizations.go

```diff
@@ -274,10 +274,14 @@ func (s *AuthorizationsService) Delete(ctx context.Context, id int64) (*Response
 // The returned Authorization.User field will be populated.
 //
 // GitHub API docs: https://developer.github.com/v3/oauth_authorizations/#check-an-authorization
```

Comment:
This URL should now be: https://developer.github.com/v3/apps/oauth_applications/#check-a-token

Also, the comment starting above at line 270 is no longer valid. Please update it.